addons/web/static/tests/views/kanban_view_tests.js

```diff
@@ -12925,4 +12925,38 @@ QUnit.module("Views", (hooks) => {
         );
         assert.containsN(target, ".o_kanban_group .o_kanban_record", 16);
     });
+    QUnit.test("Keep scrollTop when loading records with load more", async (assert) => {
+        for (let id = 1; id <= 50; id++) {
+            serverData.models.partner.records.push({
+                id,
+                bar: false,
+                display_name: `record ${id}`,
+            });
+        }
+        await makeView({
+            type: "kanban",
+            resModel: "partner",
+            serverData,
+            arch: `<kanban>
+                <templates>
+                <t t-name="kanban-box">
+                    <div style="height:250px;"><field name="id"/></div>
+                </t>
+                </templates>
+            </kanban>`,
+            groupBy: ["bar"],
+            limit: 10,
+        });
+        target.querySelector(".o_kanban_renderer").style.overflow = "scroll";
+        target.querySelector(".o_kanban_renderer").style.height = "500px";
+        const loadMoreButton = target.querySelector(".o_kanban_load_more button");
+        loadMoreButton.scrollIntoView();
+        const previous = target.querySelector(".o_kanban_renderer").scrollTop;
```

Comment:
rename previous into previousScrollTop?

Also, I would assert that the scrollTop here is not 0

Reply:
done